cairo1-run/README.md

```diff
@@ -71,6 +71,10 @@ The cairo1-run cli supports the following optional arguments:
 
 * `--append_return_values`: Adds extra instructions to the program in order to append the return and input values to the output builtin's segment. This is the default behaviour for proof_mode. Only allows `Array<felt252>` as return and input value.
 
+## Running circuits
+
+Circuits in cairo 1 require the `mod_builtin` feature to be active in order for the `AddMod`, `MulMod` and `RangeCheck96` builtins to be taken into account.
```

Comment:
I would use 'enable' instead of 'be active', to keep consistency with the [docs](https://doc.rust-lang.org/cargo/reference/features.html):

```suggestion
Circuits in cairo 1 require to enable the `mod_builtin` feature in order for the `AddMod`, `MulMod` and `RangeCheck96` builtins to be taken into account.
```

Reply:
Done!